public/public.php

```diff
@@ -966,7 +967,7 @@ public function render_image_source_box( $content = null ) {
 	 * @param int|string $id   id of the image.
 	 * @param string[]   $data metadata.
 	 * @param array      $args arguments
-	 *                         use "disable-links" = (any value)), to disable any working links.
+	 *                         use "disable-links" = (any value), to disable any working links.
```

Comment:
**Suggestion:** There seems to be a typo in the comment at line 970. The closing parenthesis is duplicated. Please correct it. [possible issue]
```suggestion
  *                         use "disable-links" = any value, to disable any working links.
```